tonic/src/transport/channel/endpoint.rs

```diff
@@ -415,6 +415,25 @@ impl Endpoint {
     pub fn uri(&self) -> &Uri {
         &self.uri
     }
+
+    /// Get the value of `TCP_NODELAY` option for accepted connections.
+    pub fn get_tcp_nodelay(&self) -> bool {
```

Comment:
Adding these makes sense to me, though we should skip the `get_()` prefixes per https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter.

Reply:
@djc I also wanted add comment like this, but the looked here: https://github.com/hyperium/tonic/blob/b4ebfc3e820dab8e4bc1b41e28bdf0965b53c441/tonic/src/transport/channel/endpoint.rs#L260.

Reply:
Fair point. I guess this is okay, then.